_data/simple-icons.json

```diff
@@ -3522,6 +3522,12 @@
             "hex": "025E8C",
             "source": "https://dependabot.com"
         },
+        {
+            "title": "Depositphotos",
+            "hex": "000000",
+            "source": "https://blog.depositphotos.com/wp-content/uploads/2020/09/depositphotos_blog_logo.svg",
+            "guidelines": "https://depositphotos.com/about/press.html"
```

Comment:
I don't see any guidelines on the usage of their logo or brand on this page so this should probably be removed.